src/convertToObject.js

```diff
@@ -12,7 +12,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
+
+  const splitedString = sourceString.split(';');
+
+  for (const item of splitedString) {
+    const splitedItem = item.split(':').map(el => el.trim());
```

Comment:
Переименуй параметры, чтобы было немного понятнее))

Вот тебе пример:
```suggestion
    const splitedItem = item.split(':').map(properties => properties.trim());
```